docs/reference/catalogs/catalogs.md

```diff
@@ -45,7 +45,8 @@ To load or unload a catalog, simply indicate in the checkbox at the left of the
 > It is not possible to have two catalogs with the same name loaded simultaneously. Attempting to do so will require one of them to be unloaded. 
 > For instance, if there is an **Environment** catalog named *MyCatalog* and a **User** catalog also named *MyCatalog*, only one of these can be loaded at a time.
 
-### Add a Catalogs
+>[!WARNING]
+> Unloading a catalog will only unload the modules where the catalog operators are defined, it will not unload third party modules that were imported during the loading of the catalog.

```

Comment:
```suggestion
>[!WARNING]
> Unloading a catalog will only unload the modules where the catalog operators are defined, it will not unload third party modules that were imported during the loading of the catalog.

### Add a Catalog
```